inductiva/fluids/fluid_tank/README.md

```diff
@@ -1,43 +1,60 @@
-# Fluid tank
+# Fluid Tank Scenario
 
-This scenario simulates the motion of a fluid in a cubic or cylindrical tank.
+The Fluid Tank Scenario simulates the motion of a single fluid in a cubic or cylindrical tank.
 Fluid is injected into the tank via an inlet located at the top of the tank and
-flows out of the tank via an outlet located at the bottom of the tank. The
+flows out of the tank via an outlet located at the bottom of the tank. The position,
+shape and size of both the inlet and the outlet can be defined by the user. The
 motion of the fluid is controlled by gravity. The simulation is performed using
 the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics)
 method.
 
 ### Example
 
-Do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
+The SPH engines available for running this scenario are DualSPHysics and SplishSplash.
 
-Initialize the scenario:
+Albeit this is a quite simple simulation scenario, the Fluid Tank Scenario illustrates the
+SPH capabilities of the API. If you would like to use other more complex SPH scenarios,
```

Comment:
I really think this does not make sense here, since scenarios are abstracted from the simulator. But I see the point of fetching more persons that desired these type of results.